packages/react-charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx

```diff
@@ -292,6 +300,14 @@ export class MultiStackedBarChartBase extends React.Component<IMultiStackedBarCh
           onMouseLeave={this._onBarLeave}
           onClick={href ? (point.placeHolder ? undefined : this._redirectToUrl.bind(this, href)) : point.onClick}
         >
+          {this.props.enableGradient && (
+            <defs>
+              <linearGradient id={`gradient_${index}_${pointData}`} >
+                <stop offset="0" stopColor={color} />
```

Comment:
call these startColor and endColor. Here and everywhere

Reply:
resolved